fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchResult.java

```diff
@@ -29,15 +28,25 @@
  */
 public class SearchResult {
     @JsonProperty
-    private PaginationInfo pagination = new PaginationInfo();
+    private PaginationInfo pagination;
     @JsonProperty
-    private List<Map<String, Object>> items = new ArrayList();
+    private List<Map<String, Object>> items;
 
     /**
-     * Default constructor
+     * Default Constructor
      */
-    public SearchResult() {
-
+    public SearchResult() {}
+    /**
+     * Constructor
+     *
+     * @param items      The individual search result items
+     * @param pagination The pagination info
+     */
+    public SearchResult(final List<Map<String, Object>> items, final PaginationInfo pagination) {
+        assert items != null;
```

Comment:
Using asserts to validate arguments is not generally considered a good practice. One common approach is something like:

`this.items = Preconditions.checkNotNull(items, "items cannot be null");`